app/views/spam2/_insights.html.erb

```diff
@@ -0,0 +1,214 @@
+<div class="card-deck w-100 mb-3">
+  <div class="card">
+    <div class="card-body">
+    <h5 class="card-title  text-secondary font-weight-bold"> <i class="fa fa-circle text-primary"></i>  Unmoderated</h5>
+      <ul class="list-group list-group-flush text-secondary ">
+        <li class="list-group-item"><%= Node.where(status: 4).count%> Nodes</li>
```

Comment:
This seems the right usage of `count` but since we are replacing it with `.size` throughout the app so maybe can do it here to achieve consistency :sweat_smile: just a suggestion

Reply:
Thanks @Tlazypanda, I will do this!! :+1: